myapp/app/views/users/show.html.erb

```diff
@@ -15,6 +15,10 @@
         <th class="w-25"><%= I18n.t(:email, scope: [:activerecord, :attributes, :user]) %></th>
         <td class="w-75"><%= @user.email %></td>
       </tr>
+      <tr>
+        <th class="w-25"><%= I18n.t(:is_admin, scope: [:activerecord, :attributes, :user]) %></th>
+        <td class="w-75"><%= @user.is_admin %></td>
+      </tr>
```

Comment:
- I changed to display users's tasks list
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/37566073/225510875-30e126ae-b63d-411a-b00f-9dffdf6b375d.png">